docs/source/open-edx-background.rst

```diff
@@ -0,0 +1,36 @@
+Open edX Background
+===================
+
+What is Open edX?
+--------------------
+Open edX refers to its two components: the Open edX Platform and the Open edX Community.
+The Open edX Platform is a scaleable and learner centered web-based platform that can be used to author and manage courses.
+This open source project is supported by an international community working together to create this multiuse platform.
+The Open edX Community consists of tCRIL, the nonprofit organization that runs the project, and a number of core and individual contributors. 
```

Comment:
Hmm. I wonder if we "run" the project. This may be a controversial word to put out there. Maybe we want "stewards"

Reply:
I took that directly from: ["The Open edX project is run by The Center for Reimagining Learning (TCRIL), a nonprofit organization (the “NP”) that was formerly known as edX Inc."](https://open.edx.org/about-open-edx/)

Reply:
🤔

Reply:
yeah OK leave as is

Reply:
Sounds good